room_with_a_view/room_with_a_view.py

```diff
@@ -1,3 +1,397 @@
+#!/usr/bin/env python3
```

Comment:
file could benefit from being broken into a few modules, but who am i to judge a hackathon?

Reply:
totally OK to say you want to do this in a subsequent PR :)

Reply:
Will do that when we run tests :-)